app/src/main/java/com/example/cattlelog/MainActivity.kt

```diff
@@ -86,15 +106,59 @@ class MainActivity : AppCompatActivity() {
             cattleList?.let { adapter.setCattleList(it) }
         })
 
+    }
 
-//        recyclerView = findViewById<RecyclerView>(R.id.herdList).apply {
-//            setHasFixedSize(true)
-//            layoutManager = viewManager
-//            adapter = viewAdapter
-//        }
+    private fun checkDatabaseVersion() {
+        var newestDBVersion = ""
+        Log.d("dropboxAPI", "entered check file function.")
+        Log.d("dropboxAPI", resources.getString(R.string.CattleLog_DropboxAPIKey))
+
+
+        val url = "https://api.dropboxapi.com/2/files/get_metadata"
+        val params = HashMap<String,String>()
+        params["path"] = "/cattlelog_database.db"
```

Comment:
Use `"/${CattlelogDatabase.DATABASE_NAME}.db"` on the right side.

Reply:
Doing so breaks the API call for some reason.  Who knows why.

Reply:
Hmm, that's weird. I tried on my end with `params["path"] = "/${CattlelogDatabase.DATABASE_NAME}.db"` and it worked.